spikingjelly/activation_based/op_counter/neuromc/core.py

```
@@ -1775,7 +1771,7 @@ def _extra_memory_for_fragment(self, fragment: _Fragment):
                     "opt_w_updated": ("FYFXKC", 32, "reg_32b", "sram_6MB"),
                 }
 
-        for _, (count_key, bits_per_elem, reg_name, sram_name) in variables.items():
+        for count_key, bits_per_elem, reg_name, sram_name in variables.values():
```

Ruff (0.16.1)

[warning] Loop control variable `reg_name` not used within loop body

Rename unused `reg_name` to `_reg_name`

(B007)